DQMOffline/Trigger/python/BPHMonitor_cfi.py

```diff
@@ -0,0 +1,54 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.bphMonitoring_cfi import bphMonitoring
+
+hltBPHmonitoring = bphMonitoring.clone()
+hltBPHmonitoring.FolderName = cms.string('HLT/BPH/Dimuon_10_Jpsi_Barrel/')
+hltBPHmonitoring.histoPSet.ptPSet = cms.PSet(
+  nbins = cms.int32 (  200  ),
+  xmin  = cms.double(   -0.5),
+  xmax  = cms.double(200),
+)
+hltBPHmonitoring.histoPSet.phiPSet = cms.PSet(
+  nbins = cms.int32 (  64  ),
+  xmin  = cms.double(   -3.2),
+  xmax  = cms.double(3.2),
+)
+hltBPHmonitoring.histoPSet.etaPSet = cms.PSet(
+  nbins = cms.int32 (  52  ),
+  xmin  = cms.double(   -2.6),
+  xmax  = cms.double(2.6),
+)
+hltBPHmonitoring.histoPSet.d0PSet = cms.PSet(
```

Comment:
1 cm ?
I think it would better to increase the resolution, why don't ou make use of a variable binning here ?